modules/ocf_puppet/manifests/puppetboard.pp

```diff
@@ -19,7 +19,7 @@
       $::hostname,
       $::fqdn,
     ],
-    proxy => 'http://lb.ocf.berkeley.edu:10009',
-    ssl   => true,
+    proxy          => 'http://lb.ocf.berkeley.edu:10009',
+    ssl            => true,
```

Comment:
Meh, ok sure

Reply:
Maybe putting this before the server aliases would make it more obvious about the lining up, but I don't think it really matters.